corehq/apps/api/odata/tests/test_feed.py

```diff
@@ -35,19 +34,15 @@ class TestCaseOdataFeed(TestCase, CaseOdataTestMixin):
     def setUpClass(cls):
         super(TestCaseOdataFeed, cls).setUpClass()
 
-        cls.client = Client()
-        cls.domain = Domain(name='test_domain')
-        cls.domain.save()
-        cls.web_user = WebUser.create(cls.domain.name, 'test_user', 'my_password')
+        cls._setupclass()
```

Comment:
nit: is a little weird to use different casing for this method (and not switch wholesale to :snake: case)

Reply:
ad4374a4472c71e8bdf8d7bac25bc00fea670736